niap-cc/Permissions/TransactIds/app/src/androidTest/java/AndroidManifest.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+ Copyright 2020 The Android Open Source Project
```

Comment:
Ultra-nit: copyright year should reflect the year the file was created for the commit.

Reply:
Fixed